c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -197,18 +197,27 @@ Version 1.6
 18. In your `apache/wsgi.conf.mako` or `apache/wsgi.conf.in` file replace all the
     `apache-entry-point` by `apache_entry_point`
 
-19. In the `{{package}}/subscribers.py`, `{{package}}/model.py`, `{{package}}/form.py` do the following changes:
+19. In your `apache/frontend.conf.mako`or `apache/frontend.conf.in` file in the
```

Comment:
mmm, I dont really like that sentence structure,
something like this would be clearer:

In your `apache/frontend.conf.mako` or `apache/frontend.conf.in` file in the
    `<LocationMatch /${instanceid}/wsgi/>` block, 
    add the following mimetypes to the existing list:

    `application/vnd.ogc.wms_xml application/vnd.ogc.gml application/vnd.ogc.se_xml`

    and in the `<LocationMatch /${vars:instanceid}/tiles/>` block,
    add the following lines:

    AddOutputFilterByType DEFLATE text/html text/plain text/xml text/css application/x-javascript text/javascript application/javascript application/xml
    Header add Access-Control-Allow-Headers "X-Requested-With, Content-Type"